packages/calcite-components/src/components/alert/alert.scss

```diff
@@ -123,14 +149,15 @@ $border-style: 1px solid var(--calcite-color-border-3);
   inset-inline: 0;
   inset-block-start: -2px;
   block-size: 2px;
-  border-radius: var(--calcite-border-radius) var(--calcite-border-radius) 0 0;
+  border-radius: var(--calcite-internal-alert-corner-radius) var(--calcite-internal-alert-corner-radius) 0 0;
```

Comment:
I think this should this be
`border-radius: var(--calcite-alert-corner-radius, var(--calcite-internal-alert-corner-radius) 0 0);`

Reply:
The dismiss progress only needs to match the top corners of the alert.